### PR TITLE
Epd implementation and update chess library

### PR DESCRIPTION
**Pull request by gahtan-syarif**
@Disservin is this what you had in mind?

### CODE REVIEW

File: src/matchmaking/match/match.cpp [hunk 1]
```diff
@@ -58,9 +58,22 @@ void Match::addMoveData(const Player& player, int64_t measured_time_ms, bool leg
     data_.moves.push_back(move_data);
 }
 
+bool isFen(const std::string line) {
```
Comment (Disservin) [inline src/matchmaking/match/match.cpp:61]: `const std::string& line`

File: src/matchmaking/match/match.cpp [hunk 2]
```diff
@@ -58,9 +58,22 @@ void Match::addMoveData(const Player& player, int64_t measured_time_ms, bool leg
...
+    // Check the number of parts
+    if (line.find(';') == std::string::npos) {
```
Comment (Disservin) [inline src/matchmaking/match/match.cpp:63]: `return line.find(';') == std::string::npos;`

**Review by Disservin — CHANGES_REQUESTED** [review]
yeah i think this should work

File: src/matchmaking/match/match.cpp (hunk 2 above)
Comment (gahtan-syarif) [reply src/matchmaking/match/match.cpp:63]: ahh right hahah

File: src/matchmaking/match/match.cpp (hunk 1 above)
Comment (gahtan-syarif) [reply src/matchmaking/match/match.cpp]: fixed 👍

File: src/matchmaking/match/match.cpp (hunk 2 above)
Comment (gahtan-syarif) [reply src/matchmaking/match/match.cpp]: fixed 👍

**Review by Disservin — APPROVED** [review]

**Pull request merged**